app/main.development.js

```diff
@@ -108,7 +108,27 @@ const installExtensions = async () => {
   }
 };
 
-const { ipcMain } = require("electron");
+const { ipcMain, systemPreferences } = require("electron");
+
+const changeDarkMode = (newAppearance) => {
+  if (newAppearance === "light" || newAppearance === "dark") {
+    globalCfg.set("theme", newAppearance === "light" ? "theme-light" : "theme-dark");
+  }
+};
+
+// Set dark / light mode on app launch
+// isDarkMode() return true or false (true when dark, false when light) so we need to convert it to 'light' or 'dark'
+if (process.platform === "darwin") {
+  changeDarkMode(systemPreferences.isDarkMode() ? "dark" : "light");
```

Comment:
Don't have a mac to test, but you should probably only do this if the theme is undefined, otherwise this will get reset every time the app is started, which might be weird if the user actually wants to run a theme other than the one for the OS.

Reply:
Good catch, will fix this

Reply:
What I meant was, this will get executed at *every* startup instead of only when the theme in decrediton is undefined (something like, `globalCfg.get("theme") === undefined`).

Can you verify whether the following happens:
- Use "systemPreferences.isDarkMode() === false" (ie, OS is configured for light mode)
- Start Decrediton
- Switch theme to dark
- Close Decrediton
- Start up again; theme reverts to light

This is what I'm questioning if it happens.

Reply:
It happens, but the theme is never undefined on startup from the first time the user starts the app

Reply:
Hmm, probably best to move this code to config initialization then. See if moving the detection and selection of the theme here works:

https://github.com/decred/decrediton/blob/master/app/config.js#L80

This should only execute when the `theme` config is not defined on the file, which will then detect and change the theme during startup only on the first run.